src/components/PluginLoader.jsx

```diff
@@ -13,9 +14,13 @@ const injectHeaderbarHidingStyles = (event) => {
         const doc = iframe.contentDocument
         const styleElement = doc.createElement('style')
         styleElement.textContent =
-            'div.app-shell-adapter header { display: none; }'
+            // adapter bar with directional styles
+            'div.app-shell-adapter > div > header,\n' +
+            // older adapter
+            'div.app-shell-adapter > header { display: none; }'
```

Comment:
Different generations of the app adapter need different selectors to hit the header bar